### PR TITLE
Killer Tomatoes Live Up to their Name

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Killer tomatoes are now hostile mobs instead of friendly mobs, thus living up to their name

Related
- Killer tomatoes are an actual fruit now
 - Activate them in your hand to turn them into mobs
- yield of regular tomatoes increased by one
- Tweaked stats on killer tomatoes to be a bit less crazy
- Killer tomatoes mutate from regular tomatoes and not blood tomatoes

:cl: Fox McCloud
tweak: Killer tomatoes actually live up to their name now and are hostile mobs
tweak: Killer tomatoes are an actual fruit now; activate it in your hands to make them into a mob
tweak: Killer tomatoes are directly mutated from regular tomatoes instead of blood tomatoes
tweak: Increased the yield of regular tomatoes by 1
tweak: Tweak some stats on the killer tomato seeds
/:cl:

### CODE REVIEW

**Comment by NTSAM** [comment]
Should draw a pair of V eyebrows at least because they look derpy and cuddly as shit

**Comment by Mijhead** [comment]
nothing could possibly go wrong :+1:

File: code/modules/hydroponics/seed_datums.dm [hunk 1]
```diff
@@ -189,15 +189,14 @@
 	name = "bloodtomato"
 	seed_name = "blood tomato"
 	display_name = "blood tomato plant"
-	mutants = list("killertomato")
```
Comment (FalseIncarnate) [inline code/modules/hydroponics/seed_datums.dm]: I'd keep it as a potential mutation of blood tomatoes, personally. Giving normal tomatoes the chance to mutate into them is fine though

**Comment by Mijhead** [comment]
I'm mildly against the loss of both self-harvest and the product being a direct mob, because those two features would make people have to think a little bit better about how to use them in a subtle way, instead of carrying a plantbag full of them.

**Comment by Fox-McCloud** [comment]
It also means you can abuse the trays and spawn 50+ hostile mobs all at the same time, instantly.

**Comment by FalseIncarnate** [comment]
The removal of self-harvest and use of the "final_form" var is actually a botanist safety thing. These tomatoes have no sense of loyalty or gratitude for their botanists, and thus will attack anyone on sight.

File: code/modules/mob/living/simple_animal/hostile/killertomato.dm [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/mob/living/simple_animal/hostile/killertomato
+	name = "Killer Tomato"
+	desc = "It's a horrifyingly enormous beef tomato, and it's packing extra beef!"
```
Comment (FalseIncarnate) [inline code/modules/mob/living/simple_animal/hostile/killertomato.dm]: Beef tomato.

**Comment by Mijhead** [comment]
Yeah, which is why I was mildly against it, but I get why it's changed.

File: code/modules/mob/living/simple_animal/hostile/killertomato.dm (hunk 2 above)
Comment (TheDZD) [inline code/modules/mob/living/simple_animal/hostile/killertomato.dm]: I honestly don't see the issue, you wouldn't say "The Grizzly bear mauled the idiot that tried to hug it."

**Pull request merged**